src/language-server/language-server.ts

```diff
@@ -89,5 +115,9 @@ export async function startLanguageServer() {
     return codeActions
   })
 
+  connection.onShutdown(() => {
```

Comment:
Should we also dispose the other subscriptions we are registering for example `connection.onInitialize`, `connection.onCodeAction` etc?